### PR TITLE
Feature/mp 7891 month db config

### PR DESCRIPTION
**Pull request by msadikkoseaupa**

### CODE REVIEW

File: src/Nox.Types.EntityFramework/Types/Month/MonthConverter.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace Nox.Types.EntityFramework.Types;
 
-public class MonthToByteConverter: ValueConverter<Month, byte>
+public class MonthConverter: ValueConverter<Month, byte>
 {
-    public MonthToByteConverter() : base(month => month.Value, monthValue => Month.From(monthValue)) { }
+    public MonthConverter() : base(month => month.Value, monthValue => Month.From(monthValue)) { }
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types.EntityFramework/Types/Month/MonthConverter.cs:7]: MonthConverter should use "FromDatabase"

File: src/Nox.Types/Types/Month/Month.cs [hunk 2]
```diff
@@ -35,4 +37,25 @@ public override string ToString()
     {
         return Value.ToString("00");
     }
```
Comment (gjorgji-grgovski-authoritypartners) [inline src/Nox.Types/Types/Month/Month.cs:39]: Can we have overloading of "From" method that accepts "int" value?

**Review by gjorgji-grgovski-authoritypartners — CHANGES_REQUESTED** [review]

File: src/Nox.Types/Types/Month/Month.cs (hunk 2 above)
Comment (msadikkoseaupa) [reply src/Nox.Types/Types/Month/Month.cs:70]: Required changes were applied

File: src/Nox.Types.EntityFramework/Types/Month/MonthConverter.cs (hunk 1 above)
Comment (msadikkoseaupa) [reply src/Nox.Types.EntityFramework/Types/Month/MonthConverter.cs]: Required changes were applied

**Review by gjorgji-grgovski-authoritypartners — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**